SFconv/SFconv.cpp

```diff
@@ -101,15 +101,15 @@ Utf8ToString(const char* s)
 	utf16::iterator out = buf;
 	utf8::const_iterator in  = s, 
 	                     end = s + len;
-    while (in != end && !(in.error() || out.error()))
```

Comment:
This appears correctly indented, the follow line (105) only being in scope of the `while`.